packages/nodes-base/nodes/SyncroMSP/SyncroMspTrigger.node.ts

```diff
@@ -0,0 +1,134 @@
+import {
+	IWebhookFunctions,
+} from 'n8n-core';
+
+import {
+	IDataObject,
+	INodeType,
+	INodeTypeDescription,
+	IWebhookResponseData,
+} from 'n8n-workflow';
+
+
+export class SyncroMspTrigger implements INodeType {
+	description: INodeTypeDescription = {
+		displayName: 'SyncroMSP Trigger',
+		name: 'syncromspTrigger',
+		icon: 'file:syncromsp.png',
+		group: ['trigger'],
+		version: 1,
+		subtitle: '={{$parameter["events"]}}',
```

Comment:
Can we parse this and if it contains the `*` we display "All events"? Right now it displays `[ * ]`.

Reply:
Should I convert the display name to 'All events' or something like this `parameter["events"].map(item => item === "*" ? "All Events" : item)` as subtitle?

Reply:
Yes, this is possible! You can also use `.join(', ')` at the end to make it a string.